syft/frameworks/torch/mpc/falcon.py

```diff
@@ -0,0 +1,82 @@
+import torch
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+from syft.workers.base import BaseWorker
+
+from typing import List
+
+
+class Falcon(object):
+    @staticmethod
+    def _share_random_tensor(
+        field: int, workers: List[BaseWorker]
+    ) -> (ReplicatedSharingTensor, ReplicatedSharingTensor):
+        """
+        TODO: Need to move this in Primitive Store:
+        Share a random tensor with the owners of some shares
+        """
+        c = torch.randint(2, size=(1,))
+        c_share_L = c.share(*workers, protocol="falcon", field=field)
+        c_share_2 = c.share(*workers, protocol="falcon", field=2)
```

Comment:
I couldn't find the secret sharing scheme you are using (can you share a link?), but if we want to be consistent with the protocol it should be  
1) replicated secret sharing , same as for `b` 
2) it should be maliciously secure secret sharing